src/pages/billing/Project.js

```diff
@@ -596,7 +603,6 @@ const ProjectDetail = ({ authorizeAndLoadAccounts, billingAccounts, billingProje
           } else {
             setTab(newTab)
           }
-          maybeLoadTotalCost()
```

Comment:
I was trying to make sure that we loaded the spend report data when the tab was first selected (we don't want to load the report when the parent page is loaded because the query costs money, and the user may never choose to view the spend report). However, this code wasn't doing what I thought because the new value of `tab` isn't actually persisted yet. Now `tab` is one of the dependencies for the effect.